pkg/openfeature/client.go

```diff
@@ -110,7 +110,15 @@ var typeToString = map[Type]string{
 type EvaluationDetails struct {
 	FlagKey  string
 	FlagType Type
-	InterfaceResolutionDetail
+	Value    interface{}
```

Comment:
The `Value` type should set in the `EvaluationDetails` struct. I'll create a dedicated issue for this. Please make this change before releasing v0.5.0